content/blogs/2023-11-15-kestra-sqs.md

```diff
@@ -0,0 +1,233 @@
+
+There's not much to it - the plugin is straightforward to use, but there are a couple of [extra parameters available](https://kestra.io/plugins/plugin-aws/tasks/sqs/io.kestra.plugin.aws.sqs.consume) if you want to dig deeper:
+
+```yaml
+id: read-from-sqs
+namespace: dev
+
+tasks:
+  - id: consumeSqsMessages
+    type: io.kestra.plugin.aws.sqs.Consume
+    maxRecords: 2
+
+
+taskDefaults:
+  - type: io.kestra.plugin.aws.sqs.Consume
+    values:
+      accessKeyId: "{{ secret('AWS_ACCESS_KEY') }}"
+      secretKeyId: "{{ secret('AWS_SECRET_KEY') }}"
+      region: "{{ secret('AWS_REGION') }}"
+      queueUrl: "{{ secret('SQS_URL') }}"
+```
+
+As earlier, running the task will redirect you to a flow execution Gantt view:
+
+![Image 11 - Flow execution Gantt view](/blogs/2023-11-15-kestra-sqs/11.png)
+
+All the good stuff is stored in the *Outputs* tab. Click on it to continue:
+
+![Image 12 - Outputs tab](/blogs/2023-11-15-kestra-sqs/12.png)
+
+You can download the consumed SQS messages locally to verify they are the ones submitted by the previous task:
+
+![Image 13 - Consumed messages](/blogs/2023-11-15-kestra-sqs/13.png)
+
+And that's pretty much all there is to the Kestra SQS integration. You can publish and consume messages, and that's actually all you need. Let's wrap things up next.
+
+
+---
+## Summing up Kestra and AWS SQS
+Today you got a 10000-foot overview of message queues and why they're essential in modern and scalable applications/APIs. You've also seen how to work with Amazon Simple Queue Service (SQS) directly from Kestra, which allows you to send messages to the queue and read them thanks to our specific plugins. By far the most common use case of this integration is to create an automated trigger flow, as shown in Example #1. This will be enough for 90% of users.
+
+To recap, queues are life savers when you want to scale your applications, and SQS is among the best queues available right now. Kestra integration allows you to integrate SQS into your new and existing workflows, which is yet another practical worth exploring.
+
+You might also be interested in other recently implemented Kestra integrations, such as the one for [SurrealDB](https://kestra.io/blogs/2023-10-09-kestra-surrealdb) and [Weaviate](https://kestra.io/blogs/2023-10-31-kestra-weaviate). Make sure to check these out, especially the latter if you want to learn how to work with **vector databases**.
+
+Subscribe to our newsletter below for more guides on modern orchestration, automation, and trends in data management, as well as new features in our platform.
+
+Join the [Slack community](https://kestra.io/slack) if you have any questions or need assistance. Follow us on [Twitter](https://twitter.com/kestra_io) for the latest news. Check the code in our [GitHub repository](https://github.com/kestra-io/kestra) and give us a star if you like the project.
```

Comment:
```suggestion
---

## Next steps

Today, you got a 10,000-foot overview of message queues and why they're so useful in modern and scalable applications. You've also seen how to work with Amazon Simple Queue Service (SQS) directly from Kestra, which allows you to react to new messages from the queue, as well as produce and consume them. By far, the most common use case of this integration is to create an automated trigger flow, as shown in Example #1. This will be enough for 90% of use cases.

To recap, queues are life savers when you want to scale your applications, and SQS is among the most mature and widely used message queues available today. 

You might also be interested in other recently implemented Kestra integrations, such as the one for [SurrealDB](https://kestra.io/blogs/2023-10-09-kestra-surrealdb) and [Weaviate](https://kestra.io/blogs/2023-10-31-kestra-weaviate). Make sure to check these out, especially the latter, if you want to learn how to work with **vector databases**.

Subscribe to our newsletter below for more guides on modern orchestration, automation, and trends in data management, as well as new features in our platform.

Join the [Slack community](https://kestra.io/slack) if you have any questions or need assistance. Follow us on [Twitter](https://twitter.com/kestra_io) for the latest news. Check the code in our [GitHub repository](https://github.com/kestra-io/kestra) and give us a star if you like the project.
```